lang/semgrep-grammars/src/semgrep-html/grammar.js

```diff
@@ -17,15 +17,51 @@ module.exports = grammar(base_grammar, {
      if they're not already part of the base grammar.
   */
   rules: {
-  /*
-    semgrep_ellipsis: $ => '...',
 
-    _expression: ($, previous) => {
+/* pad: not sure why, but this is simpler than patching element but
+   it does not work. I get some 
+
+       "The rule `start_tag` matches the empty string" tree-sitter
+
+   error.
+
+    start_tag: ($, previous) => {
+      return choice(
+       $.semgrep_start_tag,
+       ...previous.members
+      );
+    },
```

Comment:
This doesn't work because the original `start_tag` rule is not a `choice()` but a `seq()`. Here, what you want is this:
```
start_tag: ($, previous) => choice(
  $.semgrep_start_tag,
  previous
),
```